<launch>
    <arg name="vins_folder" default = "/home/davidz/work/ros/indigo/src/VINS-Mono_Ext"/>
    <arg name="config_path" default = "$(arg vins_folder)/config/tum_vio_config.yaml" />
    <arg name="vins_path" default = "$(arg vins_folder)/" />
    
    <node pkg="feature_tracker_ext" type="freak_tracker" name="freak_tracker" output="screen">
        <param name="config_file" type="string" value="$(arg config_path)" />
        <param name="vins_folder" type="string" value="$(arg vins_path)" />
	<remap from="/freak_tracker/feature_img" to="/feature_tracker_ext/feature_img"/>
    </node>

</launch>
